server/protocols/webadmin/webadmin-mailrepository/src/test/java/org/apache/james/webadmin/service/ReprocessingServiceTest.java

```diff
@@ -109,7 +112,7 @@ void reprocessingOneShouldEnqueueMail() throws Exception {
         repository.store(mail2);
         repository.store(mail3);
 
-        reprocessingService.reprocess(PATH, KEY_2, new ReprocessingService.Configuration(SPOOL, NO_TARGET_PROCESSOR, CONSUME, Limit.unlimited()));
+        reprocessingService.reprocess(PATH, KEY_2, new ReprocessingService.Configuration(SPOOL, NO_TARGET_PROCESSOR, Optional.empty(), CONSUME, Limit.unlimited()));
```

Comment:
Have a constant for `Optional.empty()` say `UNLIMITED_MAX_RETRIES`  would make the tests more readable IMO.